agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -69,6 +69,8 @@ class main_thread : public btl::transport_socket_thread {
     bool handle_1905_autoconfiguration_response(ieee1905_1::CmduMessageRx &cmdu_rx,
                                                 const std::string &src_mac);
     bool handle_1905_discovery_query(ieee1905_1::CmduMessageRx &cmdu_rx);
+    bool handle_1905_ap_capability_query(ieee1905_1::CmduMessageRx &cmdu_rx);
```

Comment:
not needed for this stage, remove

Reply:
I might be missing out something but I'm pretty sure this is just the header for the `main_thread` function that we're using to log.